.github/workflows/build_image.yml

```diff
@@ -37,9 +37,10 @@ jobs:
             type=semver,pattern={{major}}.{{minor}}
 
       - name: Build and push Docker image 🏗️
-        uses: docker/build-push-action@v4
+        uses: docker/build-push-action@v6
         with:
-          push: ${{ github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/releases') }}
+          # push: ${{ github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/releases') }}
+          push: true
```

Comment:
This will release a new docker image upon every single push to every single branch. Is that what you really wanted?